devfile.yaml

```diff
@@ -0,0 +1,24 @@
+schemaVersion: 2.1.0
+metadata:
+  name: python-hello-world
+components:
+  - name: python
+    container:
+      image: quay.io/eclipse/che-python-3.8:next
```

Comment:
could you avoid `next` tag as it may break this sample

Also you could give a try on `quay.io/devfile/base-developer-image:ubi8-7bd4fe3`

Reply:
Replaced with `quay.io/devfile/base-developer-image:ubi8-7bd4fe3`